packages/axios-extension/test/abap/abap-adt-service.test.ts

```diff
@@ -120,7 +122,7 @@ describe('Create new transport number', () => {
             .replyWithFile(200, join(__dirname, 'mockResponses/transportRequest-1.txt'));
         const transportRequestService = await provider.getAdtService<TransportRequestService>(TransportRequestService);
         expect(
-            await transportRequestService.createTransportRequest({
+            await transportRequestService?.createTransportRequest({
```

Comment:
Why use the optional chaining operator? When could `transportRequestService` be falsy? Is it an error in that case?

Reply:
The assertion in all the 4 test cases expect the output to be array or at least empty array. If transportRequestService is null, the actual output with '?' would be undefined, and thus fail the expect assertion.